src/write.rs

```diff
@@ -431,14 +431,30 @@ where
     /// Remove the given value from the value-bag of the given key.
     ///
     /// The updated value-bag will only be visible to readers after the next call to `refresh()`.
+    #[deprecated(since = "11.0.0", note = "Renamed to remove_value")]
     pub fn remove(&mut self, k: K, v: V) -> &mut Self {
+        self.remove_value(k, v)
+    }
+
+    /// Remove the given value from the value-bag of the given key.
+    ///
+    /// The updated value-bag will only be visible to readers after the next call to `refresh()`.
+    pub fn remove_value(&mut self, k: K, v: V) -> &mut Self {
         self.add_op(Operation::Remove(k, v))
```

Comment:
Should I also rename these operations? I imagine those are internal, implementation details and should be safe to rename? Although maybe not valuable ⚖️

Reply:
Yes, I think renaming them is probably worthwhile.
I forget whether `Operation` is `pub` actually.. :/

Reply:
Yeah it looks like `Operation` is `pub`. So I imagine we want to deprecate them instead of a straight rename? That means that we have to match on deprecated enum variants in a few places. Is that okay? Or do you want me to just delete those? And do I need to do any sort of "cargo ignore this match of a deprecated enum" so it's not super annoying?

Reply:
I think what I want to do is make `Operation` not be `pub`. That is a breaking change I think I'd be willing to make for `11.0`. If you change it to `pub(crate)`, how many things break?

Reply:
It looks like maybe just [`pending`](https://github.com/jonhoo/rust-evmap/blob/master/src/write.rs#L361)?

Reply:
Yeah, let's remove `pending` too. That's not really an API people should be using anyway.

Reply:
This is a dream come true. Love deleting code.

Reply:
Oh wait, you just meant make not public, I assume? Poor me 😢

Reply:
So for [this test](https://github.com/jonhoo/rust-evmap/blob/master/tests/lib.rs#L188) do I remove the assertion and just put a dummy `assert!(true)` after the `flush()` or do I move the test into `src/write`?

Reply:
I think you can actually just delete it. I don't think we use it internally for anything?
For that test, I'd move it to be a unit test, and then it can inspect the internal state that `pending()` accesses directly.

Reply:
So dfb4843184f9c8aa3e974e8607f753138ef5d53a makes `Operation` `pub(crate)`, removes `pending()`, and moves the test